seatunnel-dist/release-docs/NOTICE

```diff
@@ -4367,6 +4367,17 @@ The Apache Software Foundation (http://www.apache.org/).
 
 =========================================================================
 
+jmockdata NOTICE
+
+========================================================================
+jmockdata
+  * LICENSE:
+    * license/LICENSE-jmockdata.txt (Apache License 2.0)
+  * HOMEPAGE:
+    * https://github.com/jsonzou/jmockdata
+
+========================================================================
+
```

Comment:
It seems jmockdata doesn't have a NOTICE file in their repo.
https://github.com/jsonzou/jmockdata

Reply:
OK, my understanding of notice is wrong.
And something exception in e2e, I resolve it later